### ISSUE TITLE
upgrade vagrant image to use elasticsearch2

### ISSUE THREAD

**Issue by missinglink** [pelias/vagrant#50]
upgrade vagrant image to use elasticsearch2

it's likely that this image will be broken since the `elasticsearch2` upgrade was merged to the `master` branches.

**Comment by pmezard** [comment]
Hello,

I am trying to fix this, and started a bit before @orangejulius published his PR for nodejs. Two questions:
- Should I keep going? I knew next to nothing about vagrant/chef before starting, so @orangejulius cleanup is likely better than mine, and upgrading ES after that does not seem difficult.
- If so, is there any test suite I can run against the vagrant setup to be confident it still works (a little)? I am completely new to pelias codebase, and started with vagrant to be able to play with it.

**Comment by orangejulius** [comment]
Hey @pmezard, thanks for offering to help! Definitely give it a shot if you have the time, and even if you get stuck let us know how it goes. Feel free to share incomplete/nearly working code for feedback here. We also have a chat room at https://gitter.im/pelias/pelias if that's easier. Perhaps together we can get it working relatively easily!

**Comment by pmezard** [comment]
As I mentioned in another ticket, I am using pelias/api ciao tests to validate the upgrade (and fix the tests too). Do you know any other test suite I could use in addition to api/ciao or is it good enough?

**Comment by orangejulius** [comment]
Those should be sufficient, but if you did happen to import any data in particular, you can compare queries on the vagrant box to those running on our hosted instance: feel free to use http://pelias.io/compare to run some queries

**Comment by orangejulius** [comment]
Fixed by https://github.com/pelias/vagrant/pull/56, although there are other issues causing the Vagrant image still to not work great, which will be addressed separately.
